packages/parser-html/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "dependencies": {
+    "jsdom": "^12.0.0"
+  },
+  "description": "webhint parser needed to analyze HTML files",
+  "devDependencies": {
+    "@types/cheerio": "^0.22.9",
+    "@types/debug": "0.0.30",
+    "@types/jsdom": "^11.0.6",
+    "@types/node": "8.0.14",
```

Comment:
Remove types that are not necessary.

(we try to keep them to a minimum, because they often break our builds 😔)

Reply:
Also, after you do these changes, can you run `yarn` in the root so that `yarn.lock` file gets updated. Thanks!